[Bug] 3.4.0 beta 无法启用 forkTSChecker

## What happens?

配置项启用 forkTSChecker 无法正常启用插件

## Mini Showcase Repository(REQUIRED)

https://github.com/urielvan/umi-fork-ts-bug

## How To Reproduce

执行 `npm run start` 之后控制台中输出了错误信息，src/pages/index.tsx 中参数 `foo` 为隐式 any 类型，正常情况下应该被报错

```
internal/modules/cjs/loader.js:968
  throw err;
  ^

Error: Cannot find module 'D:\Code\fork-ts\node_modules\@umijs\deps\compiled\fork-ts-checker-webpack-plugin\TypeScriptReporterRpcService.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)

√ Webpack
  Compiled successfully in 3.14s

RpcIpcMessagePortClosedError: Process 3860 exited with code "1" [null]
Issues checking service aborted - probably out of memory. Check the `memoryLimit` option in the ForkTsCheckerWebpackPlugin configuration.
If increasing the memory doesn't solve the issue, it's most probably a bug in the TypeScript or EsLint.

```

## Context

- **Umi Version**: 3.4.0-beta6
- **Node Version**: 12.18.3
- **Platform**: windows 10

--- a/packages/bundler-webpack/package.json
+++ b/packages/bundler-webpack/package.json
@@ -34,7 +34,7 @@
   },
   "dependencies": {
     "@umijs/bundler-utils": "3.4.0-beta.6",
-    "@umijs/deps": "0.2.29",
+    "@umijs/deps": "0.2.30",
     "@umijs/types": "3.4.0-beta.6",
     "@umijs/utils": "3.4.0-beta.6",
     "jest-worker": "26.6.2",

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -26,7 +26,7 @@
   "dependencies": {
     "@umijs/ast": "3.4.0-beta.6",
     "@umijs/babel-preset-umi": "3.4.0-beta.6",
-    "@umijs/deps": "0.2.29",
+    "@umijs/deps": "0.2.30",
     "@umijs/utils": "3.4.0-beta.6"
   }
 }

--- a/packages/preset-built-in/package.json
+++ b/packages/preset-built-in/package.json
@@ -28,7 +28,7 @@
     "@types/react-router-config": "5.0.1",
     "@umijs/babel-preset-umi": "3.4.0-beta.6",
     "@umijs/bundler-webpack": "3.4.0-beta.6",
-    "@umijs/deps": "0.2.29",
+    "@umijs/deps": "0.2.30",
     "@umijs/renderer-mpa": "3.4.0-beta.6",
     "@umijs/renderer-react": "3.4.0-beta.6",
     "@umijs/runtime": "3.4.0-beta.6",

--- a/packages/server/package.json
+++ b/packages/server/package.json
@@ -24,7 +24,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@umijs/deps": "0.2.29",
+    "@umijs/deps": "0.2.30",
     "@umijs/utils": "3.4.0-beta.6"
   }
 }

--- a/packages/types/package.json
+++ b/packages/types/package.json
@@ -27,7 +27,7 @@
   "dependencies": {
     "@umijs/babel-preset-umi": "3.4.0-beta.6",
     "@umijs/core": "3.4.0-beta.6",
-    "@umijs/deps": "0.2.29",
+    "@umijs/deps": "0.2.30",
     "@umijs/server": "3.4.0-beta.6",
     "@umijs/utils": "3.4.0-beta.6",
     "webpack-chain": "6.5.1"

--- a/packages/umi/package.json
+++ b/packages/umi/package.json
@@ -30,7 +30,7 @@
   "dependencies": {
     "@umijs/bundler-webpack": "3.4.0-beta.6",
     "@umijs/core": "3.4.0-beta.6",
-    "@umijs/deps": "0.2.29",
+    "@umijs/deps": "0.2.30",
     "@umijs/preset-built-in": "3.4.0-beta.6",
     "@umijs/runtime": "3.4.0-beta.6",
     "@umijs/types": "3.4.0-beta.6",

--- a/packages/utils/package.json
+++ b/packages/utils/package.json
@@ -24,6 +24,6 @@
     "access": "public"
   },
   "dependencies": {
-    "@umijs/deps": "0.2.29"
+    "@umijs/deps": "0.2.30"
   }
 }

